physics/cu_gf_deep.F90

```diff
@@ -190,41 +196,47 @@ subroutine cu_gf_deep_run(        &
   ! omega (omeg), windspeed (us,vs), and a flag (ierr) to turn off
   ! convection for this call only and at that particular gridpoint
   !
-     real(kind=kind_phys),    dimension (its:ite,kts:kte)                              &
+     real(kind=kind_phys),    dimension (its:ite,kts:kte)              &
         ,intent (in   )                   ::                           &
         dhdt,rho,t,po,us,vs,tn
 !$acc declare copyin(dhdt,rho,t,po,us,vs,tn)
-     real(kind=kind_phys),    dimension (its:ite,kts:kte)                              &
+     real(kind=kind_phys),    dimension (its:ite,kts:kte)              &
         ,intent (inout   )                ::                           &
         omeg
 !$acc declare copy(omeg)
-     real(kind=kind_phys),    dimension (its:ite,kts:kte)                              &
+     real(kind=kind_phys),    dimension (its:ite,kts:kte)              &
         ,intent (inout)                   ::                           &
          q,qo,zuo,zdo,zdm
 !$acc declare copy(q,qo,zuo,zdo,zdm)
-     real(kind=kind_phys), dimension (its:ite)                                         &
+     real(kind=kind_phys), dimension (its:ite)                         &
         ,intent (in   )                   ::                           &
         dx,z1,psur,xland
 !$acc declare copyin(dx,z1,psur,xland)
-     real(kind=kind_phys), dimension (its:ite)                                         &
+     real(kind=kind_phys), dimension (its:ite)                         &
         ,intent (inout   )                ::                           &
         mconv,ccn
 !$acc declare copy(mconv,ccn)
+     real(kind=kind_phys), dimension (its:ite,kts:kte,nchem)           &
+        ,intent (inout)                   ::                           &
+        chem3d
+     logical, intent (in) :: do_smoke_transport
+     real(kind=kind_phys), dimension (its:ite,nchem)                   &
+         , intent (out) :: wetdpc_deep
+     real(kind=kind_phys), intent (in) :: fscav(:)
 
```

Comment:
@haiqinli I think you might need to add open ACC directives for the new fields?
@timsliwinski-noaa Would you mind reviewing this PR for GPU compliance?

Reply:
@dustinswales Sure, I will add open ACC directories for these new fields.

Reply:
@danielabdi-noaa Updated following your suggestions. Thanks.